src/kirby/components/list/list.component.ts

```diff
@@ -49,24 +43,27 @@ export class ListComponent implements OnInit {
 
   isSectionsEnabled: boolean;
   isSelectable: boolean;
+  isLoading: boolean;
+  isLoadOnDemandEnabled: boolean;
 
-  constructor() {}
+  constructor(private listHelper: ListHelper) {}
 
-  ngOnInit() {
+  ngOnInit(): void {
```

Comment:
We don't put return type `void` on methods
```suggestion
  ngOnInit() {
```

Reply:
Does that go for all functions or just this *ngOnInit* ?

Reply:
I have removed `void` from my void functions.